Content.Server/GameObjects/Components/Construction/ConstructorComponent.cs

```diff
@@ -49,8 +51,7 @@ void TryStartStructureConstruction(GridCoordinates loc, string prototypeName, An
 
             var transform = Owner.Transform;
 
-            var interactionSystem = _entitySystemManager.GetEntitySystem<InteractionSystem>();
-            if (!interactionSystem.InRangeUnobstructed(loc.ToMap(_mapManager), Owner.Transform.WorldPosition, ignoredEnt: Owner, insideBlockerValid: prototype.CanBuildInImpassable))
+            if (!EntitySystem.Get<SharedInteractionSystem>().InRangeUnobstructed(loc.ToMap(_mapManager), Owner.Transform.WorldPosition, ignoredEnt: Owner, insideBlockerValid: prototype.CanBuildInImpassable))
```

Comment:
Why doesn't this use `InteractionChecks`?

Reply:
Because all the InteractionChecks expect event args which aren't present here, but I will add another method that can be used here.